modules/workspaces/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "tfe_workspace" "this" {
+  name         = var.name
+  organization = var.organization
+
+  auto_apply            = var.auto_apply
+  file_triggers_enabled = var.file_triggers_enabled
+  operations            = var.operations
+  queue_all_runs        = var.queue_all_runs
+  ssh_key_id            = var.ssh_key_id
+  terraform_version     = var.terraform_version
+  trigger_prefixes      = var.trigger_prefixes
+  working_directory     = var.working_directory
+
+  dynamic "vcs_repo" {
+    for_each = lookup(var.vcs_repo, "identifier", "void") == "void" ? [] : [var.vcs_repo]
```

Comment:
```suggestion
    for_each = lookup(var.vcs_repo, "identifier", null) == null ? [] : [var.vcs_repo]
```

Reply:
This has some implications beyond just this line, although the logic here precedes my work on this module. I'll have to check w/ the either John or Hugo to see why `void` was chosen instead of `null`.

Reply:
@osterman @aknysh Is it okay if we keep this as is for now?

Reply:
it should work either way, it's just a marker (we used `null` in other modules) (unless it has some implications)